README.md

```diff
@@ -33,72 +33,50 @@ We recommend [installing GOV.UK Frontend using node package manager
-| Android          | Samsung Internet (latest 2 versions)   | compliant   |
+You should also use:
 
-‘Compliant’ means that the components must look as good and function as well as
-they do in other modern browsers.
+- [the JavaScript from GOV.UK Frontend](https://github.com/alphagov/govuk-frontend/blob/master/docs/installation/installing-with-npm.md#using-javascript)
+- [a separate stylesheet](https://github.com/alphagov/govuk-frontend/blob/master/docs/installation/supporting-internet-explorer-8.md) if you support Internet Explorer 8
 
-'Functional' means the components may not look perfect in the given browser, but
-must still be usable without errors and without 'looking broken'.
+Your service will not meet level AA of WCAG 2.1 if you use [compatibility mode](https://github.com/alphagov/govuk-frontend/blob/master/docs/installation/compatibility.md).
 
-If you are including GOV.UK Frontend as part of a stylesheet that you are
-generating in your application's build pipeline, you will need to [generate and
-include a separate stylesheet in order to support Internet Explorer
-8](docs/installation/supporting-internet-explorer-8.md).
+### Browser and assistive technology support
 
-[service-manual-browsers]: https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices#browsers-to-test-in
+GOV.UK Frontend supports the Government Digital Service’s recommended [browsers](https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices#browsers-to-test-in) and [assistive technologies](https://www.gov.uk/service-manual/technology/testing-with-assistive-technologies#what-to-test).
 
-## Assistive technology support
+GOV.UK Frontend also:
 
-GOV.UK Frontend will allow you to build services that comply with the [guidance
-in the Service Manual][service-manual-assistive-technologies].
+- is accessible for users who only use a keyboard
+- lets users adjust or override colours in Windows, Firefox and Chrome
+- is usable in Internet Explorer 8, 9 and 10
 
-Currently, GOV.UK Frontend officially supports the following assistive technologies:
+### Accessibility issues and warnings
 
-| Software                 | Version        | Type               | Browser                   |
-|--------------------------|----------------|--------------------|---------------------------|
-| JAWS                     | 15 or later    | screen reader      | Internet Explorer 11      |
-| ZoomText                 | 10.11 or later | screen magnifier   | Internet Explorer 11      |
-| Dragon NaturallySpeaking | 11 or later    | speech recognition | Internet Explorer 11      |
-| NVDA                     | 2016 or later  | screen reader      | Firefox (latest versions) |
-| VoiceOver                | 7.0 or later   | screen reader      | Safari on iOS10 and OS X  |
+We're working on fixing the following issues from our [issues list in GitHub](https://github.com/alphagov/govuk-frontend/issues/).
```

Comment:
I'm in two minds because it feels like it might be helpful for users to have a concise summary list. It'd be good to see what the issues list would look like - are you able to create the label so I can see what the user journey would be?

Reply:
@m-green As an example, here's link for the the issues with the label "accessibility": https://github.com/alphagov/govuk-frontend/labels/accessibility

Reply:
Ok - I think this could work (for this first list), if we can add the label Nick mentioned so that users see only issues that do not meet WCAG 2.1 AA. 
Perhaps we could also tweak the titles of the GitHub issues so they're as easy to possible to understand in the list alone?

Reply:
Yeah I think it's really important that we focus the content where the workflow of the team is.

If we don't we'll not keep the README up-to-date, so even if it may be easier to scan the summary it'll be incorrect.

Reply:
Definitely see where you're coming from, but that approach can lead us to prioritise our needs (eg ease of updating or time to update) over the user need for plain language. We can always put content processes in place to ensure we keep guidance content in sync with changes.

All ok in this instance though! Are you or Hanna ok to split everything out and add the appropriate labels? Then I'll add the link here before we merge.

Reply:
@hannalaakso mentioned at standup that she should be able to split out into separate issues - I'll update everything else we discussed in the meantime.

Reply:
I've tried retroactively applying a 'accessibility-regulations-failure' label: 

https://github.com/alphagov/govuk-frontend/labels/accessibility-regulations-failure

As you can see there are no open issues, I may have missed some of the closed ones too.